docs/source/reference/style_transfer.rst

```diff
@@ -0,0 +1,35 @@
+##############
+Style Transfer
+##############
+
+********
+The task
+********
+
+TODO
+
```

Comment:
Mind describing the task ?

Reply:
link to here would be nice too: https://paperswithcode.com/task/style-transfer

Reply:
@edgarriba see https://github.com/PyTorchLightning/lightning-flash/pull/262#discussion_r631697099